### PR TITLE
Updates to BasicViews

### PR DESCRIPTION
**Pull request by rynowak**
- Remove DNX
- Add EF + LocalDB
- Make Layout more like our template

Completes the requirements for building out https://github.com/aspnet/Mvc/issues/4175

### CODE REVIEW

File: testapp/BasicViews/Startup.cs [hunk 1]
```diff
@@ -34,6 +58,16 @@ public void Configure(IApplicationBuilder app)
...
 
+        private void CreateDatabase(IServiceProvider services)
+        {
+            using (var serviceScope = services.GetRequiredService<IServiceScopeFactory>().CreateScope())
+            {
+                var dbContext = services.GetRequiredService<BasicViewsContext>();
+                dbContext.Database.EnsureDeleted();
+                dbContext.Database.EnsureCreated();
+            }
+        }
```
Comment (rynowak) [inline testapp/BasicViews/Startup.cs]: Is this the best way to set this up? I want the database aspect to "just work" as much as possible.

**Comment by rowanmiller** [comment]
EF stuff looks good to me

File: testapp/BasicViews/Startup.cs (hunk 1 above)
Comment (rowanmiller) [inline testapp/BasicViews/Startup.cs]: Yep, this seems fine - to me this is legitimately part of configuring your environment

**Comment by rynowak** [comment]
Thanks. Going to self :shipit: myself and check this in.

**Pull request merged**